src/devices/Dhtxx/DhtSensor.cs

```diff
@@ -173,15 +173,31 @@ private bool ReadThroughOneWire()
             // pull up data line
             _controller.Write(_pin, PinValue.High);
             // wait 20 - 40 microseconds
-            DelayHelper.DelayMicroseconds(40, true);
+            DelayHelper.DelayMicroseconds(30, true);
```

Comment:
Is this wait or timing offset specified in the datasheet? If so it would be great to use the value they provide.

Reply:
Waiting time is between 20 - 40 microseconds. If I write 40 microseconds in the code, the actual waiting time will be greater than 40 microseconds. The extra microseconds will wait in the next step loop.

Reply:
I see that makes sense. I don't have the hardware here to test it, but given that seems like the current implementation wasn't really working and this seems to be fixed for you, I'll go ahead and merge this.